Contributing.md

```diff
@@ -1,42 +1,83 @@
+
+The following example shows a sample script for `Range.getValue`:
+
+```YAML
+'ExcelScript.Range#getValue:member(1)':
+    - |-
+    /**
+     * This sample reads the value of A1 and prints it to the console.
+     */
+    function main(workbook: ExcelScript.Workbook) {
+        // Get the current worksheet.
+        let selectedSheet = workbook.getActiveWorksheet();
+    
+        // Get the value of cell A1.
+        let range = selectedSheet.getRange("A1");
+        
+        // Print the value of A1.
+        console.log(range.getValue());
+    }
+```
 
-**Note**: Most of the information in this section can be found in [GitHub Help](https://docs.github.com/) articles. If you're familiar with Git and GitHub, skip to the [Adding samples](#adding-samples) section for the specifics of the code/content flow of this repository.
+Please add samples to the **.yaml** file in alphabetical order, based on the class and method names. This makes them easier to maintain.
 
-Use GitHub to contribute to this documentation without having to clone the repo to your desktop. This is the easiest way to create a pull request in this repository. It's the recommended way to add a single sample.
+### What makes a good sample?
 
-> **Important**: The reference content in this repository is autogenerated. Pull requests to this repo should either add samples or address tooling or other infrastructure issues.
+A good sample has the following characteristics:
 
-### To contribute using GitHub
+1. Descriptive - The sample has an introductory comment explaining the behavior and comments in the code as needed.
+1. Useful - Some or all of the sample could be copied into an actual script and quickly modified to suit someone's needs.
+1. Readable - The sample uses simple TypeScript, effective whitespace, and meaningful variable names.
+1. Targeted - The sample showcases a specific API or scenario and only includes enough other code to be functional.
 
-1. Find the article you want to contribute to on GitHub (usually the [Excel Scripts Samples](/docs/sample-scripts/excel-scripts.yaml)).
+### When do samples get added to the actual documentation pages?
+
+After the **sample-scripts** files are updated, a member of the team will run the documentation tooling and regenerate the API pages. Your new sample is added then.
+
+### Contribute using GitHub
+
+Use GitHub to contribute to this documentation without having to clone the repo to your desktop. This is the easiest way to create a pull request in this repository. Use this method to make a minor change that doesn't involve code changes.
+
+Using this method allows you to contribute to one article at a time.
+
+#### To contribute using GitHub
+
+1. Find the article you want to contribute to on GitHub.
 2. Once you are on the article in GitHub, sign in to GitHub (get a free account [Join GitHub](https://github.com/join)).
```

Comment:
nit: for consistency, can all the ordered lists be formatted similarly (e.g. all 1's or 123)?